ui/app/pages/create-account/connect-hardware/select-hardware.js

```diff
@@ -211,24 +190,45 @@ export default class SelectHardware extends Component {
     )
   }
 
-  renderFooter() {
-    return (
-      <div className="hw-connect__footer">
-        <h3 className="hw-connect__footer__title">
-          {this.context.t('readyToConnect')}
-        </h3>
-        {this.renderButtons()}
-        <p className="hw-connect__footer__msg">
-          {this.context.t('havingTroubleConnecting')}
+  renderTrezorTutorialSteps() {
+    const steps = [
+      {
+        asset: 'plug-in-wallet',
+        dimensions: { width: '225px', height: '75px' },
+        title: this.context.t('step1TrezorWallet'),
+        message: this.context.t('step1TrezorWalletMsg', [
+          // eslint-disable-next-line react/jsx-key
           <a
-            className="hw-connect__footer__link"
-            href="https://support.metamask.io/"
-            target="_blank"
+            className="hw-connect__msg__link"
+            href="https://metamask.zendesk.com/hc/en-us/articles/360020394612-How-to-connect-a-Trezor-or-Ledger-Hardware-Wallet"
             rel="noopener noreferrer"
+            target="_blank"
           >
-            {this.context.t('getHelp')}
-          </a>
-        </p>
+            click here
+          </a>,
+        ]),
+      },
+    ]
+
+    return (
+      <div
+        className="hw-tutorial"
+        ref={(node) => {
+          this.referenceNode = node
+        }}
+      >
+        {steps.map((step, index) => (
+          <div className="hw-connect" key={index}>
+            <h3 className="hw-connect__title">{step.title}</h3>
+            <p className="hw-connect__msg">{step.message}</p>
+            <img
+              className="hw-connect__step-asset"
+              src={`images/${step.asset}.svg`}
```

Comment:
I saw the following request in Chrome, and I think this line may be the culprit:

`chrome-extension://oehplhbehlbgjdapfkoemdgjlgbjpkkg/images/undefined.svg`